rpcd/playbooks/roles/rpc_support/tasks/holland_preinstall.yml

```diff
@@ -24,6 +38,9 @@
 - name: Install pip dependencies
   pip:
     name: "{{ item }}"
+    state: latest
+    virtualenv: "{{ holland_venv_enabled | bool | ternary(holland_venv|dirname, omit) }}"
```

Comment:
You're going to have to add a slash at the end of holland_venv. Other wise the dirname filter will wack out the first entry of the path, leaving you with "/openstack/venvs"